packages/composer/test/proxy.test.js

```diff
@@ -46,112 +46,42 @@ test('should increment and decrement activeWsConnections metric', async t => {
   const { service, wsServer } = await createWebsocketService(t)
   wsServer.on('connection', socket => {
     socket.on('message', message => {
-      socket.send(message)
+      setTimeout(() => {
+        socket.send(message)
+      }, 500)
```

Comment:
why?

Reply:
Locally this test was very flaky. Sometimes things progressed before prom metrics were updated.